### PR TITLE
Phpunit to phpt v1.2

### PR DESCRIPTION
**Pull request by bjori**

### CODE REVIEW

File: tests/generic/mongoint32_32bit-001.phpt [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 --SKIPIF--
+<?php require __DIR__ . "/skipif.inc" ?>
 <?php if (4 !== PHP_INT_SIZE) { die('skip Only for 32-bit platform'); } ?>
 --FILE--
 <?php
-$mongo = new Mongo('mongodb://localhost');
-$coll = $mongo->selectCollection('test', 'mongoint32');
+require_once __DIR__ ."/../utils.inc";
+$mongo = mongo();
+$coll = $mongo->selectCollection('phpunit', 'mongoint32');
```
Comment (jmikola) [inline tests/generic/mongoint32_32bit-001.phpt:10]: Why do we want to use a `phpunit` database?

**Comment by jmikola** [comment]
Did you forget to add `skipif.inc` to `mongotimestamp-tostring.phpt` and `mongotimestamp-error.phpt`?

File: tests/generic/mongoint32_32bit-001.phpt (hunk 1 above)
Comment (bjori) [inline tests/generic/mongoint32_32bit-001.phpt:10]: Blah.
Good point. It should be using the dbname() function.

In a authenticated environment there are no guarantees that we have access to that databse, so it needs to be configured in mongo-test-cfg.inc

**Comment by bjori** [comment]
Yep. skipif was forgotten in those two

**Pull request merged**